### PR TITLE
Facebook login

### PR DESCRIPTION
**Pull request by AvishMadan**
Hi ketan
I have updated one class where fix facebook login. I have added one new testcase for forgot password.

### CODE REVIEW

File: src/feb/CodeAddedInNewBranch.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package feb;
+
+public class CodeAddedInNewBranch {
+	public static void main(String[] args) {
+		
+		System.out.println("I am working as new employee in new branch");
+		
+	}
```
Comment (aishwaryaparange) [inline src/feb/CodeAddedInNewBranch.java:8]: Can you please add one more syso system

File: src/feb/CodeAddedInNewBranch.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package feb;
+
+public class CodeAddedInNewBranch {
+	public static void main(String[] args) {
+		
+		System.out.println("I am working as new employee in new branch");
+
```
Comment (Patil135) [inline src/feb/CodeAddedInNewBranch.java:7]: I am not happy with this change please add new syso statement.

File: src/feb/CodeAddedInNewBranch.java (hunk 1 above)
Comment (RK5031860) [inline src/feb/CodeAddedInNewBranch.java:8]: i am not happy with this code

File: src/feb/CodeAddedInNewBranch.java (hunk 2 above)
Comment (Subodhra) [inline src/feb/CodeAddedInNewBranch.java:7]: I am not happy with your code can you write one syso statement

Comment (123jps) [inline src/feb/CodeAddedInNewBranch.java:7]: can you please add one more line

File: src/feb/CodeAddedInNewBranch.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package feb;
+
+public class CodeAddedInNewBranch {
+	public static void main(String[] args) {
+
```
Comment (MansiDave) [inline src/feb/CodeAddedInNewBranch.java:5]: can you add one syso stmt

File: src/feb/CodeAddedInNewBranch.java (hunk 1 above)
Comment (RK5031860) [reply src/feb/CodeAddedInNewBranch.java:8]: code

File: src/feb/CodeAddedInNewBranch.java (hunk 2 above)
Comment (AvishMadan) [reply src/feb/CodeAddedInNewBranch.java:7]: I have made the changes as per your comment

**Review by Subodhra — APPROVED** [review]

**Review by Patil135 — APPROVED** [review]

**Review by aishwaryaparange — APPROVED** [review]

**Review by 123jps — APPROVED** [review]

**Pull request merged**